sdk/core/azure-core/test/ut/sha_test.cpp

```diff
@@ -7,38 +7,114 @@
 
 using namespace Azure::Core::Cryptography::_internal;
 
+// cspell: words ABCDE FGHIJ
```

Comment:
Can you check formating on this file.  Appears to be an extra space of indentation

Reply:
clang-format seems to be fine with it.

Where do you see the extra indentation?